core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -225,18 +233,37 @@ var TopicEditorPage = function() {
     expect(await subtopics.count()).toEqual(count);
   };
 
-  this.addSubtopic = async function(title) {
-    await addSubtopicCard.click();
+  this.addSubtopic = async function(title, imgPath, htmlContent) {
+    await addSubtopicButton.click();
     await newSubtopicTitlefield.sendKeys(title);
+
+    await workflow.submitImage(
+      topicThumbnailButton, thumbnailContainer, imgPath, false);
+    var subtopicPageContentButton = element(by.css(
+      '.protractor-test-show-schema-editor'));
+    await waitFor.elementToBeClickable(subtopicPageContentButton,
+      'Edit subtopic htm content button taking too long to be clickable');
+    await subtopicPageContentButton.click();
+
+    var pageEditor = element(by.css(
+      '.protractor-test-create-subtopic-page-content'));
+    await waitFor.visibilityOf(pageEditor,
+      'Subtopic html editor takes too long to appear');
+    await (await browser.switchTo().activeElement()).sendKeys(htmlContent);
```

Comment:
Have you checked that calling `sendKeys` directly on the element doesn't work? Calling `sendKeys` directly would be more readable if that works

Reply:
Yeah it was failing.